tidy3d/web/api/container.py

```diff
@@ -701,8 +711,19 @@ def pbar_description(task_name: str, status: str) -> str:
             "postprocess",
             "visualize",
             "success",
-        ]
-        end_statuses = ("success", "error", "errored", "diverged", "diverge", "deleted", "draft")
+        )
+
+        end_statuses = (
+            "success",
+            "error",
+            "errored",
```

Comment:
remove errored, diverge, deleted?(do we still have deleted status), abort (won't use it any more), to keep the code clear